en/components/grids_and_lists.md

```diff
@@ -0,0 +1,199 @@
+      <a href="/components/grid_virtualization.html" class="feature__more-details">Virtualization Directive Walkthrough</a>
+    </div>
+    <div class="feature__details">
+      <h3 id="virtualization-and-performance"><a href="/components/grid_virtualization.html">Virtualization and Performance</a></h3>
+      <p>The updated Ignite UI for Angular Virtualization directive ensures that you can seamlessly scroll through large-scale data sets with millions of points, giving our fastest Angular Data Grid high-speed rendering and no-lag loading.</p>
+    </div>
+  </div>
+
+  <div class="feature">
+    <div class="feature__image feature__image--left">
+      <img class="b-lazy" src="https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-lazy-load.png" data-src="https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-paging-768.gif" data-srcset="https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-paging-480.gif 480w, https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-paging-768.gif 768w, https://static.infragistics.com/marketing/ignite-ui-angular/grid/ignite-ui-angular-grid-paging-1100.gif 1100w" sizes="(min-width: 1800px) 575px,(max-width: 1799px) and (min-width: 1200px) 50vw,(max-width: 1199px) and (min-width: 992px) 33vw,(max-width: 991px) and (min-width: 768px) 50vw, 100vw" alt="Paging component allows for programmatic pagination">
+      <a href="/components/grid.html" class="feature__more-details">Paging Component Documentation</a>
```

Comment:
If you have class to the anchor, please use the following syntax - `<a href="grid.md" class="feature__more-details">Paging Component Documentation</a>`